utils/swift_build_support/swift_build_support/products/product.py

```diff
@@ -293,7 +293,7 @@ def target_for_platform(self, platform, arch, include_version=True):
                 arch, self.args.darwin_deployment_version_xros)
         return target
 
-    def generate_darwin_toolchain_file(self, platform, arch):
+    def generate_darwin_toolchain_file(self, platform, arch, version=None):
```

Comment:
(Optional) It may make sense to state the intent of the `version` parameter so not to "overload" it with additional meanings in the future
```suggestion
    def generate_darwin_toolchain_file(self, platform, arch, macos_deployment_version=None):
```